src/main/java/seedu/jarvis/logic/commands/Command.java

```diff
@@ -8,6 +8,13 @@
  */
 public abstract class Command {
 
+    /**
+     * Gets the command word of the command.
+     *
+     * @return {@code String} representation of the command word.
+     */
+    public abstract String getCommandWord();
```

Comment:
Alternative: can define `COMMAND_WORD` as a `protected final` field in `Command`, then have all sub-classes define their own `COMMAND_WORD` via the constructor, so you don't have to define `getCommandWord()` for every class.

Reply:
Doesn't work for static fields, and the static field in each command class is useful for `JarvisParser` and testing.